Sources/SQLKit/Builders/SQLUnionBuilder.swift

```diff
@@ -0,0 +1,21 @@
+public final class SQLUnionBuilder: SQLQueryBuilder {
+    public var query: SQLExpression {
+        return self.union
+    }
+
+    public var union: SQLUnion
+    public var database: SQLDatabase
+
+    public init(on database: SQLDatabase,
+                all: Bool = false,
+                _ args: [SQLSelectBuilder]) {
+        self.union = .init(all: all, args.map(\.select))
+        self.database = database
+    }
+}
+
+extension SQLDatabase {
+    public func union(all: Bool = false, _ args: SQLSelectBuilder...) -> SQLUnionBuilder {
```

Comment:
That wasn't what I meant 😅 I was thinking more this:

```swift
public func union(_ args: SQLSelectBuilder...) -> SQLUnionBuilder
public func union(all args: SQLSelectBuilder...) -> SQLUnionBuilder
```
The different labels are sufficient to differentiate the overloads.

Reply:
Ah gotcha! If you think that's clear enough as a distinction - I like it!

Reply:
At the very least, it's a pattern Tanner and I have used repeatedly elsewhere 🤷‍♀️

Reply:
Who's Tanner? 😈